go/kbfs/kbfsgit/runner.go

```diff
@@ -502,11 +504,45 @@ func (r *runner) printStageStart(ctx context.Context,
 	r.needPrintDone = true
 }
 
+func (r *runner) printGitJournalStart(ctx context.Context) {
+	adj := "encrypted"
+	if r.h.Type() == tlf.Public {
+		adj = "signed"
+	}
+	if r.verbosity >= 1 {
+		r.printStageStart(ctx,
+			[]byte(fmt.Sprintf("Syncing %s data to Keybase: ", adj)),
+			"mem.flush.prof", "")
+	}
+}
+
+func (r *runner) printGitJournalMessage(
+	ctx context.Context, lastByteCount int, totalSize, sizeLeft int64) int {
+	const bytesFmt string = "(%.2f%%) %s... "
+	eraseStr := strings.Repeat("\b", lastByteCount)
+	flushed := totalSize - sizeLeft
+	if flushed < 0 {
+		flushed = 0
```

Comment:
When is this expected to happen?

Reply:
It's not really expected.  I swear I saw a negative number once when I was testing this out, but I couldn't reproduce it and I couldn't figure out how it was possible.  So this is just a safeguard.